### ISSUE TITLE
The README mentions -mod=readonly - but -mod is not an option

### ISSUE THREAD

**Issue by rayjlinden** [golangci/golangci-lint#437]
In the README there is a section that talks about the --mod flag.  It would be useful in a CI situation to not modify the go.mod file - so I wanted to use -mod=readonly.  However, in the latest version of golangci-lint this option does not appear to be available.  At the very least the README is wrong.  Does the feature indeed exist?  Is it named something else?

```
$ golangci-lint --version
golangci-lint has version 1.15.0 built from 901cf25 on 2019-02-18T08:22:26Z
$ golangci-lint run -mod=readonly
ERRO Can't get config for command line: can't parse args: unknown shorthand flag: 'm' in -mod-readonly 
```

**Comment by emilyruby** [comment]
The mention of `-mod` is in reference to the `modules-download-mode` option in your config file (`.golangci.yml`). It is just describing how the option is used in the background.

You should set `modules-download-mode` to `readonly` in the config rather than pass it into the command line.

**Comment by dobs** [comment]
Looks like it was intentionally removed, see https://github.com/golangci/golangci-lint/issues/339

I'd also be interested in having this re-added. Sounds like the removal was intentional, but happy to put up a PR to re-add it otherwise.

**Comment by jirfag** [comment]
Hi!
Are you talking about this part of example yaml config?
```yaml
 # by default isn't set. If set we pass it to "go list -mod={option}". From "go help modules":
  # If invoked with -mod=readonly, the go command is disallowed from the implicit
  # automatic updating of go.mod described above. Instead, it fails when any changes
  # to go.mod are needed. This setting is most useful to check that go.mod does
  # not need updates, such as in a continuous integration and testing system.
  # If invoked with -mod=vendor, the go command assumes that the vendor
  # directory holds the correct copies of dependencies and ignores
  # the dependency descriptions in go.mod.
```
?

I didn't find any mentions of `-mod` in README except that.

**Comment by rayjlinden** [comment]
Yep

**Comment by jirfag** [comment]
In this example config I was talking about `go list` flag
